iceoryx_posh/include/iceoryx_posh/internal/runtime/ipc_interface_base.hpp

```diff
@@ -42,8 +45,15 @@
 #include <process.h>
 #endif
 
+
 namespace iox
 {
+namespace platform
+{
+#if defined(_WIN32)
+using IoxIpcChannelType = iox::posix::NamedPipe;
+#endif
+} // namespace platform
```

Comment:
I think the platform can be removed when we define `IoxIpcChannelType` for non-windows platforms here

Reply:
@elBoberido As this is a platform-based thing I would prefer to keep the namespace `platform`. This clearly states the intent.